cms/templates/widgets/header.html

```diff
@@ -1,4 +1,3 @@
-<%page expression_filter="h"/>
```

Comment:
Reviewers: All fixes in this file were reverted because it had one of the more major issues (i.e. 2 page tags), and the original implementor did not have time now for correcting and re-testing.